hoomd/md/many_body.py

```diff
@@ -0,0 +1,402 @@
+            * ``n`` (`float`) - :math:`n` - power in :math:`b_{ij}` (dimensionless, *default*: 0.0)
+            * ``gamma`` (`float`) - :math:`\\gamma` - coefficient in :math:`b_{ij}` (dimensionless, *default*: 0.0)
+            * ``c`` (`float`) - :math:`c` - coefficient in :math:`g(\\theta)` (dimensionless, *default*: 0.0)
+            * ``d`` (`float`) - :math:`d` - coefficient in :math:`g(\\theta)` (dimensionless, *default*: 1.0)
+            * ``m`` (`float`) - :math:`m` - coefficient in :math:`g(\\theta)` (dimensionless, *default*: 0.0)
+
+    Example::
+
+        nl = md.nlist.Cell()
+        tersoff = md.many_body.Tersoff(r_cut=1.3, nlist=nl)
+        tersoff.params[('A', 'B')] = dict(magnitudes=(2.0, 1.0), lambda3=5.0)
+    """
+    _cpp_class_name = "PotentialTersoff"
+    def __init__(self, nlist, r_cut=None, r_on=0., mode='none'):
+        super().__init__(nlist, r_cut, r_on, mode);
+        params = TypeParameter(
+                'params',
+                'particle_types',
+                TypeParameterDict(cutoff_thickness=0.2,
+                    magnitudes=(1.0, 1.0),
+                    exp_factors=(2.0, 1.0),
+                    lambda3=0.0,
+                    dimer_r=1.5,
+                    n=0.0,
+                    gamma=0.0,
+                    c=0.0,
+                    d=1.0,
+                    m=0.0,
+                    alpha=3.0,
+                    len_keys=2)
+                )
+        self._add_typeparam(params)
+
+
+class RevCross(Triplet):
+    """Reversible crosslinker three-body potential to model bond swaps.
+
+    Args:
+        nlist (:py:mod:`hoomd.md.nlist`): Neighbor list
+        r_cut (float): Default cutoff radius (in distance units).
+        r_on (float): Default turn-on radius (in distance units).
+        mode (str): Energy shifting mode.
+
+    :py:class:`RevCross` specifies that the revcross three-body potential
+    should be applied to every non-bonded particle pair in the simulation.
+    Despite the fact that the revcross potential accounts for the effects of
+    third bodies, it is actually just a combination of two body potential terms.
+    It can thus use type-pair parameters similar to those of the pair potentials.
+
+    The RevCross potential has been described in detail in
+    `S. Ciarella and W.G. Ellenbroek 2019 <https://arxiv.org/abs/1912.08569>`_.
+    It is based on a generalized Lennard-Jones pairwise attraction to form bonds
+    between interacting particles:
+
+    .. math::
+        :nowrap:
+
+        \\begin{eqnarray*}
+        V_{ij}(r)  =  4 \\varepsilon \\left[ \\left( \\dfrac{ \\sigma}{r_{ij}} \\right)^{2n}
+        - \\left( \\dfrac{ \\sigma}{r_{ij}} \\right)^{n} \\right] \\qquad r<r_{cut}
+        \\end{eqnarray*}
+
+    with the following coefficients:
+
+    - :math:`\\varepsilon` - *epsilon* (in energy units)
+    - :math:`\\sigma` - *sigma* (in distance units)
+    - :math:`n` - *n* (unitless)
+    - :math:`m` - *m* (unitless)
```

Comment:
Remove. There is no `m` parameter.

Reply:
Actually, remove this whole list. Look at the `md.pair` documentation style to see how we label the math symbols for the parameters only in one place in the typeparam documentation below.